java/cdk/src/main/java/sleeper/cdk/SleeperCdkApp.java

```diff
@@ -102,6 +104,12 @@ public class SleeperCdkApp extends Stack {
     private PersistentEmrBulkImportStack persistentEmrBulkImportStack;
     private EksBulkImportStack eksBulkImportStack;
     private QueryQueueStack queryQueueStack;
+    private AutoStopEcsClusterTasksStack autoStopEcsClusterTasksStack;
+    private AutoDeleteS3ObjectsStack autoDeleteS3ObjectsStack;
+    private LoggingStack loggingStack;
+    private Boolean generateAutoDeleteS3ObjectsStack = true;
+    private Boolean generateLoggingStack = true;
+    private Boolean generateProperties = true;
```

Comment:
We could do with a comment explaining what these flags are for, so that we can control when those stacks are deployed in the system test CDK app.